packages/babel-plugin-polyfill-regenerator/src/index.ts

```diff
@@ -39,3 +48,7 @@ export default defineProvider<Options>(({ debug }, options) => {
 
 const isRegenerator = meta =>
   meta.kind === "global" && meta.name === "regeneratorRuntime";
+
+function shallowEqual(obj1: any, obj2: any) {
+  return JSON.stringify(obj1) === JSON.stringify(obj2);
```

Comment:
This is obviously a pretty naive implementation of shallow equality, but it has no deps and is _very_ short. Let me know if that's okay or if you'd prefer something less naive

Reply:
I don't like it, but it's ok in this case since it's comparing very small serializable objects.

Reply:
Yep that was kind of my thought process too 👍